schemas/Activity

```diff
@@ -51,7 +51,7 @@
             "rdfs:comment": "A description of the item.",
             "rdfs:label": "description",
             "schema:domainIncludes": {
-                "@id": "schema:Thing"
+                "@id": "reproschema:Activity"
```

Comment:
you should not change any property that you are directly borrowing from schema.org

Reply:
so replace any schema.org property with their objects.